app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Here is information about an user</h1>
+<%= render 'partials/user_card', user: @user %>
+<div class='user-bio'>
+  <h3>Bio</h3>
+  <h3><%= @user.bio %></h3>
+</div>
+<ul class='thre-posts-list'>
+  <% @posts.each_with_index do |post, index| %>
+    <li><%= render 'partials/post_card', post: post, index: index %></li>
+  <% end %>
+</ul>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: Good job adding most of all the changes mentioned in the previous review. I also suggest you apply this approach in this page(`app/views/users/show.html.erb`) as you did for other pages by making sure when clicking on each `Post` takes you to its show page as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB3P7JH5WRW2BYO7YOFFAM6YY6QWGQA). 
**Congratulations :tada: :tada: **